### PR TITLE
add about/index.html and make site about me

### PR DESCRIPTION
**Pull request by oksana-liseczkaya**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <p>Так как я уже <mark>более двух лет</mark> работаю менеджером в продуктовой IT-компании, очень часто приходится принимать участие в<br/> разработке нового продукта. Хочу углубить свои знания в этом напралении и быть полезной в этой сфере. </p>
+<hr/>
+<h3>&#128170; Мои сильные качества разработчика</h3>
+<p>Умение работать в <ins>любой</ins> обстановке</p>
+<p>Готовность к исследованиям</p>
+<p>Коммуникационные навыки</p>
+<p>Способность планировать</p>
+<hr/>
+<p><a href="mailto:liseczkaya@gmail.com" target="_blank">Моя почта</a></p>
+<p><a href="https://github.com/oksana-liseczkaya" target="_blank">Мой профиль в GitHub</a></p>
```
Comment (ghost) [inline about/index.html]: Please add indent from 18 to 26 line. It will be more prettier.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline index.html]: `meta` tags should be closed.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline about/index.html]: `meta` tags should be closed.

File: about/index.html (hunk 1 above)
Comment (oksana-liseczkaya) [reply about/index.html]: add indent from 18 to 26 line

File: about/index.html (hunk 3 above)
Comment (oksana-liseczkaya) [reply about/index.html]: make meta tags closed

File: index.html (hunk 2 above)
Comment (oksana-liseczkaya) [reply index.html]: make meta tags closed

**Pull request merged**